watertap/unit_models/electrocoagulation.py

```diff
@@ -0,0 +1,991 @@
+            initialize=0.001,
+            bounds=(0, 0.1),
+            units=pyunits.m,
+            doc="Electrode thickness",
+        )
+
+        self.electrode_mass = Var(
+            initialize=10,
+            bounds=(0, None),
+            units=pyunits.kg,
+            doc="Electrode mass",
+        )
+
+        self.electrode_volume = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.m**3,
+            doc="Electrode volume",
+        )
+
+        self.cathode_area = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.m**2,
+            doc="Area of cathode",
+        )
+
+        self.anode_area = Var(
+            initialize=1,
+            bounds=(1e-6, None),
+            units=pyunits.m**2,
+            doc="Area of anode",
+        )
+
+        self.electrode_gap = Var(
+            initialize=0.005,
+            bounds=(0.00001, 0.5),
+            units=pyunits.m,
+            doc="Electrode gap",
+        )
+
+        self.electrolysis_time = Var(
+            initialize=30,
+            bounds=(0.1, 200),
+            units=pyunits.minute,
+            doc="Electrolysis time",
+        )
+
+        self.cell_volume = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.m**3,
+            doc="Volume of electrolytic cell",
+        )
+
+        self.current_density = Var(
+            initialize=1,
+            bounds=(0, 2000),
+            units=pyunits.ampere / pyunits.m**2,
+            doc="Current density",
+        )
+
+        self.applied_current = Var(
+            initialize=1e4,
+            bounds=(0, None),
+            units=pyunits.ampere,
+            doc="Applied current",
+        )
+
+        self.ohmic_resistance = Var(
+            initialize=0.1,
+            bounds=(0, None),
+            units=pyunits.ohm * pyunits.m**2,
+            doc="Ohmic resistance of solution",
+        )
+
+        self.charge_loading_rate = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.coulomb / pyunits.liter,
+            doc="Charge loading rate",
+        )
+
+        self.current_efficiency = Var(
+            initialize=1,
+            bounds=(0, 5),
+            units=pyunits.kg / pyunits.kg,
+            doc="Current efficiency",
+        )
+
+        self.cell_voltage = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.volt,
+            doc="Cell voltage",
+        )
+
+        self.overpotential = Var(
+            initialize=1,
+            bounds=(0, None),
+            units=pyunits.volt,
+            doc="Overpotential",
+        )
+
+        @self.Expression(doc="Conductivity")
+        def conductivity(b):
+            return pyunits.convert(
+                prop_in.conc_mass_phase_comp["Liq", "TDS"] / b.tds_to_cond_conversion,
+                to_units=pyunits.S / pyunits.m,
+            )
```

Comment:
Do you think we should eventually add this to MCAS?

Reply:
Yep. If I was an experimentalist with a cond > TDS parameter that I knew worked for my system, I would want to have this option.